sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+Write-Warning "Certificates created by this script MUST NOT be used for production."
+Write-Warning "They expire after 30 days, and most importantly are provided for demonstration purposes to help you quickly understand CA Certificates."
+Write-Warning "When productizing against CA Certificates, you'll need to use your own security best practices for certification creation and lifetime management."
+
+# Check to make sure openssl is installed
+if (-Not (Get-Command "openssl" -ErrorAction SilentlyContinue)) {
+  throw "openssl is not availabe. You will need to install it or add it to the PATH to proceed"
+}
+
+# Remove previous cert stores
+if (Test-Path device_cert_store.pem) {
+  Remove-Item device_cert_store.pem
+}
+
+#Wrap commands to catch errors
+function Invoke-IoTCommand {
+
+  param(
+    [string]$Command = $(throw "Must provide executable command string")
+  )
+
+  Invoke-Expression "$Command"
+
+  if ($LastExitCode -ne 0) {
+    throw "'$Command' invocation failed with exit code ($LastExitCode)"
+  }
+}
+
+Invoke-IoTCommand -Command 'openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey'
+Invoke-IoTCommand -Command 'openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"'
```

Comment:
You're setting `days 365` but at top of file you say it expires in 30 days.  Are these the same things?

Reply:
good point those are different. updating